docs/specs/legacy.yml

```diff
@@ -390,7 +390,6 @@ outputs:
     }
 ---
 # Orphan file, with multiple toc of same site path, look up by file path(sub then parent)
-os: windows
```

Comment:
why is this working on all platforms now?

Reply:
Test-case's been updated